src/test/java/bio/terra/pdao/bigquery/BigQueryPdaoTest.java

```diff
@@ -78,10 +78,10 @@ private Study makeStudy(String studyName) {
         tables.add(table1);
         tables.add(table2);
 
-        Study study = new Study()
-                .setName(studyName)
-                .setDescription("this is a test study")
-                .setTables(tables);
+        Study study = new Study();
+        study.setName(studyName)
```

Comment:
Curious what motivated that change. Seems like it didn't change anything.

Reply:
yes, i needed to make this change and i don't like it. the setName and setDescription are defined on StudySummary and they return a StudySummary so I can't chain Study setters to them. I can call setTables first and then the StudySummary methods, but i can't return that object because the signature is returning a Study.
suggestions welcome!